superset/sql_parse.py

```diff
@@ -41,6 +42,16 @@
 logger = logging.getLogger(__name__)
 
 
+# Workaround for https://github.com/andialbrecht/sqlparse/issues/652.
+sqlparse.keywords.SQL_REGEX.insert(
+    0,
+    (
+        re.compile(r"'(''|\\\\|\\|[^'])*'", re.IGNORECASE | re.UNICODE).match,
```

Comment:
Just in case, can we use the default flags from `sqlparse` to make sure the compiled regex is in line with those in the library, something like
```suggestion
        re.compile(r"'(''|\\\\|\\|[^'])*'", sqlparse.keywords.FLAGS).match,
```